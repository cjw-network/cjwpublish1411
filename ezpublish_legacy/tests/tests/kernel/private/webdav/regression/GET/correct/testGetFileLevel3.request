<?php

// Test if GET returns correct information on the file $GLOBALS['ezc_webdav_testfolder']/file.jpg

// First upload the file as 'file.jpg' in the folder $GLOBALS['ezc_webdav_testfolder']
$article = new ezpObject( 'image', (int)$GLOBALS['ezc_webdav_testfolderid'] );
$article->name = 'file';
$article->image = dirname( __FILE__ ) . DIRECTORY_SEPARATOR . $GLOBALS['ezc_webdav_testfolder'] . '.jpg';
$article->publish();

$_SERVER['REQUEST_URI'] = '/' . $GLOBALS['ezc_siteaccess'] . '/Content/' . $GLOBALS['ezc_webdav_testfolder'] . '/file.jpg';
$_SERVER['REQUEST_METHOD'] = 'GET';

?>
