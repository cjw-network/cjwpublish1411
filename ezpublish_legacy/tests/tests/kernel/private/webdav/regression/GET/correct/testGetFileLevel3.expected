ETag: ab747e616eebbbda42bf10e5a6fc10fa
Server: eZComponents/dev/ezcWebdavTransportWrapper
Content-Type: image/jpeg; charset="utf-8"
Content-Length: 57049
HTTP/1.1 200 OK
